frontend/src/features/public-form/PublicFormService.ts

```diff
@@ -164,6 +170,48 @@ export const submitStorageModeForm = async ({
   ).then(({ data }) => data)
 }
 
+export const submitStorageModeFormV2 = async ({
```

Comment:
suggestion - create a fetch version as well; our longer term plan is to handle the fetch fallback more elegantly, but this can be done in another PR

Reply:
fetch version created in [d6e9db3](https://github.com/opengovsg/FormSG/pull/6665/commits/d6e9db359c1e494a792b28388408643b8bf8eb45)